test/pg_rls/admin_test.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class PgRlsAdminTest < ActiveSupport::TestCase
+  teardown do
+    PgRls.reset_config!
+  end
+
+  test "admin_execute configures the connection to use the admin shard" do
+    PgRls.stub :connects_to, { shards: { admin: { writing: :admin, reading: :admin } } } do
+      result = PgRls::Admin.admin_execute { ::ActiveRecord::Base.connection_db_config.name }
+      assert_equal "primary", result
+    end
+  end
+end
```

Comment:
this is great, but can we add some more test to it? like select of record / update so it would ensure the core functionality please, thanks